utils/rocvideodecode/roc_video_dec.cpp

```diff
@@ -52,6 +52,7 @@ RocVideoDecoder::RocVideoDecoder(int device_id, OutputSurfaceMemoryType out_mem_
 
 
 RocVideoDecoder::~RocVideoDecoder() {
+    std::chrono::_V2::system_clock::time_point start_time = StartTimer();
```

Comment:
you can use `auto` instead of `std::chrono::_V2::system_clock::time_point` here.